bin/setup

```diff
@@ -1,28 +1,22 @@
 #!/usr/bin/env ruby
-require 'pathname'
 require 'fileutils'
-include FileUtils
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+APP_ROOT = File.expand_path('..', __dir__)
 
 def system!(*args)
   system(*args) || abort("\n== Command #{args} failed ==")
 end
 
-chdir APP_ROOT do
-  # This script is a starting point to setup your application.
+FileUtils.chdir APP_ROOT do
+  # This script is a way to setup or update your development environment automatically.
+  # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
   # Add necessary setup steps to this file.
 
   puts '== Installing dependencies =='
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
-
   puts "\n== Preparing database =="
   system! 'bin/rails db:setup'
```

Comment:
`system! 'bin/rails db:prepare'`

can you also update this line according to [this][1]

[1]: http://railsdiff.org/5.2.4.3/6.0.3.2